src/resolve/resolved-etc-hosts.c

```diff
@@ -381,6 +381,21 @@ static int manager_etc_hosts_read(Manager *m) {
         return 1;
 }
 
+static int answer_add_ptr(DnsAnswer *answer, DnsResourceKey *key, const char *name) {
+
```

Comment:
No empty line before variable defs.

Reply:
The guideline is to leave an empty line when the function declaration spans multiple lines and to not leave an empty line if the function declaration fits on a single line, so in this case, there's no need for an empty line